src/multi-level-checkbox-editor/multi-level-checkbox-editor.less

```diff
@@ -1,155 +1,160 @@
-
-            .node-list-item {
-                @node-padding();
-
-                display: flex;
-                justify-content: space-between;
-                align-items: center;
-                position: relative;
-                cursor: pointer;
-                white-space: pre-line;
-
-                &.highlighted {
-                    background: @info-light;
-                }
-
-                div {
-                    display: flex;
-                    align-items: center;
-
-                    label {
-                        margin-bottom: 0;
-                    }
-                }
-
-                .child-node-count {
-                    display: flex;
-                    align-items: center;
-                    justify-content: center;
-                    background: @gray-light;
-                    min-width: 55px;
-                    padding: 5px;
-                    border-radius: 10%;
-                }
-            }
+        .child-node-count {
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          background: @gray-light;
+          min-width: 55px;
+          padding: 5px;
+          border-radius: 10%;
         }
+      }
     }
+  }
 }
 
 .multi-level-checkbox-editor-filter {
-    margin-left: 12px;
-    width: 275px;
-    height: 32px;
-    border: @default-border;
-    .search-input {
-        width: initial;
-    }
+  margin-left: 12px;
+  width: 275px;
+  height: 32px;
+  border: @default-border;
+
+  .search-input {
+    width: initial;
+  }
 }
 
 .info-box {
-    max-width: 400px;
-    display: flex;
-    margin-top: 20px;
+  max-width: 400px;
+  display: flex;
+  margin-top: 20px;
 
-    &-icon {
-        font-size: 14px;
-        padding-right: 10px;
-    }
+  &-icon {
+    font-size: 14px;
+    padding-right: 10px;
+  }
 
-    &.info {
-        color: @info-deep;
-        .info-box-icon {
-            @info-icon-inv();
-        }
-    }
+  &.info {
+    color: @info-deep;
 
-    &.warning {
-        color: @warning;
-        margin-left: 20px;
-        .info-box-icon {
-            @warning-icon-inv();
-        }
+    .info-box-icon {
+      @info-icon-inv();
     }
-}
+  }
 
-.empty-nodebox {
-    display: flex;
-    margin: 10px 12px;
-    width: 180px;
-    border: 1px solid #D7D7D7;
-    background: #FFFFFF;
-    padding: 20px;
-    max-height: 250px;
-    box-shadow: 0 1px 1px rgba(0, 0, 0, 0.05);
+  &.warning {
+    color: @warning;
+    margin-left: 20px;
 
-    > p {
-        margin: auto;
-        text-align: left;
+    .info-box-icon {
+      @warning-icon-inv();
     }
+  }
 }
 
+.empty-nodebox {
+  display: flex;
+  margin: 10px 12px;
+  width: 180px;
+  border: 1px solid @gray-mid;
+  background: @white;
+  padding: 20px;
+  max-height: 250px;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.05);
+
+  .empty-nodebox-content {
+    margin: auto;
+    text-align: left;
+  }
+}
 
+.arrange-type-box {
+  width: 300px ! important;
```

Comment:
Please avoid important rules. They spread very easily and are hard to control.

Reply:
done removed it